### PR TITLE
Updated Fastlane for better rendering & removed wear screenshots

### PR DESCRIPTION
**Pull request by MagicLike**
I updated the full-description according to the requested changes from @izzysoft (https://github.com/jamie-mh/AuthenticatorPro/pull/576#issuecomment-1418268926) and removed the wear screenshots & promo image, due to F-Droid incompatability.

### CODE REVIEW

**Review by IzzySoft — COMMENTED** [review]
LGTM, thanks!

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -5,36 +5,24 @@ It supports TOTP (Time Based) and HOTP (Counter Based) authenticators using eith
...
 - 🎨 **Customisation**: Set icons and rename. You can also arrange your authenticators in any order you like so you can find them easily.
-
 - ⌚ **Wear OS**: Quickly view your authenticators directly from your watch. Please note that a connection to your Android device is required. Furthermore, F-Droid distributions of the app are not compatible with Wear OS, since non-free dependencies are required.
 
+(¹) *To scan QR-Codes, the app requires the camera permission.*
 
-(*) *To scan QR-Codes, the app requires the camera permission.*
-
-
-**Frequently Asked Questions:** https://github.com/jamie-mh/AuthenticatorPro/wiki#frequently-asked-questions
```
Comment (IzzySoft) [inline fastlane/metadata/android/en-US/full_description.txt:34]: -> **<a href='https://github.com/jamie-mh/AuthenticatorPro/wiki#frequently-asked-questions'>FAQ</a>**

**Review by IzzySoft — COMMENTED** [review]
Sorry, almost missed: HTML, not Markdown. While both are supported by my repo, F-Droid.org doesn't support Markdown. You might also want to convert the `**bold**` to `<b>bold</b>`.

**Comment by MagicLike** [comment]
> Sorry, almost missed: HTML, not Markdown. While both are supported by my repo, F-Droid.org doesn't support Markdown. You might also want to convert the `**bold**` to `<b>bold</b>`.

Are HTML header also supported?

**Comment by IzzySoft** [comment]
> Are HTML header also supported?

In my repo they'll get removed (or rather replaced by a `<p>` with a style applied to make it bold; see [here](https://gitlab.com/IzzyOnDroid/repo/-/blob/master/lib/upstream.class.php#L875) for details); app descriptions must not interfere with the page structure (headers are not for optical formatting but to structure the page). At F-Droid I saw only a single app using them, and it looked horrible – so not sure if they are stripped there now as well.

**Comment by MagicLike** [comment]
> > Are HTML header also supported?
> 
> In my repo they'll get removed (or rather replaced by a `<p>` with a style applied to make it bold; see [here](https://gitlab.com/IzzyOnDroid/repo/-/blob/master/lib/upstream.class.php#L875) for details); app descriptions must not interfere with the page structure (headers are not for optical formatting but to structure the page). At F-Droid I saw only a single app using them, and it looked horrible – so not sure if they are stripped there now as well.

And what about the <hr> tag?

**Comment by IzzySoft** [comment]
Not part of the [forbidden HTML tags](https://gitlab.com/fdroid/fdroidserver/-/blob/master/fdroidserver/lint.py#L205) at fdroidserver. Not part of the allowed tags in mine either, as it was never needed (so it would be simply removed here), but I see no harm in adding it if it's really needed.

**Comment by MagicLike** [comment]
> Not part of the [forbidden HTML tags](https://gitlab.com/fdroid/fdroidserver/-/blob/master/fdroidserver/lint.py#L205) at fdroidserver. Not part of the allowed tags in mine either, as it was never needed (so it would be simply removed here), but I see no harm in adding it if it's really needed.

The I will just leave it ^^

**Comment by MagicLike** [comment]
Please review @jamie-mh

**Comment by IzzySoft** [comment]
> The I will just leave it 

Fine with me. Looking at your changes, I see no problem with my updater removing them either :stuck_out_tongue_winking_eye: Let's see what @jamie-mh has to say about it. LGTM so far, though I didn't "count letters" again :see_no_evil:

**Comment by jamie-mh** [comment]
Looks good to me. Thanks!

**Comment by MagicLike** [comment]
> Looks good to me. Thanks!

No problem!

**Comment by MagicLike** [comment]
@jamie-mh Ooops I made a small mistake... Fixed here: #653

**Comment by IzzySoft** [comment]
No need to take the colon outside. I usually leave it inside as well :wink:

**Comment by MagicLike** [comment]
I just wanted to make it equal with the current version... 😅

**Comment by IzzySoft** [comment]
I leave the decision to @jamie-mh – it doesn't hurt, but is not strictly required. A cosmetic question I guess…

**Pull request merged**